differential.go

```diff
@@ -66,3 +66,20 @@ func (c *Conn) DifferentialGetCommitMessage(
 
 	return &res, nil
 }
+
+// DifferentialRevisionSearchMethod is method name on Phabricator API.
+const DifferentialRevisionSearchMethod = "differential.RevisionSearch"
+
+// DifferentialRevisionSearch performs a call to differential.revision.search.
+func (c *Conn) DifferentialRevisionSearch(
+	req requests.DifferentialRevisionSearchRequest,
+) (*responses.DifferentialRevisionSearchResponse, error) {
+	var res responses.DifferentialRevisionSearchResponse
+
+	if err := c.Call(
+		DifferentialRevisionSearchMethod, &req, &res); err != nil {
```

Comment:
is it github display change or does it look funky, like `DifferentialRevisionSearchMethod` is part of block?  Maybe extra newline to make it more clear?